### ISSUE TITLE
Please support OAuth

### ISSUE THREAD

**Issue by stapelberg** [frab/frab#20]
Hey,

instead of creating the 5000th account, I’d like to use OAuth.

Thanks in advance,
Best regards,
Michael

**Comment by zonque** [comment]
Yes, frab should really be restructured to use devise instead of its own, home-brew authentication solution.

**Comment by manno** [comment]
devise is now part of frab (again). I'd also like to see OAUTH, but I feel like there is still not enough testing around authentication/user editing. But with the new ["feature" (capybara/poltergeist) tests](https://github.com/frab/frab/blob/master/test/features/can_login_test.rb) that's easy to add.
